### DISCUSSION TITLE
issue in my github page

### DISCUSSION THREAD

**Discussion by albeeralkhawri** [community/community#77838] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hi my webiste working localy fine but when push to github i get js file lodnig error 

did contat yours on 1/12/2023 about same issue no one back to me this is the first refrence number #77588 can you have a look and help as my website effect and i can't go live 

Best Regards,
Albeer Alkhawri

**Comment by ColinHuelsbusch** [comment]
Hi, the files are not linked correctly. 

The images and JavaScript files are in the "software-city" subfolder not in the root folder.

https://albeeralkhawri.github.io/software-city/images/*.png not https://albeeralkhawri.github.io/images/*.png 
https://albeeralkhawri.github.io/software-city/js/*.js not https://albeeralkhawri.github.io/js/*.js 

You have to remove ../ before the images and ../../ before the JavaScript files.

Kind regards,
Colin
